aptos-move/aptos-vm/src/counters.rs

```diff
@@ -3,20 +3,24 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use aptos_metrics_core::{
-    exponential_buckets, register_histogram, register_int_counter, register_int_counter_vec,
+    histogram_opts, register_histogram, register_int_counter, register_int_counter_vec,
     register_int_gauge, Histogram, IntCounter, IntCounterVec, IntGauge,
 };
 use once_cell::sync::Lazy;
 
+const BLOCK_EXECUTION_TIME_BUCKETS: [f64; 17] = [
+    0.20, 0.30, 0.40, 0.50, 0.60, 0.70, 0.80, 0.90, 1.0, 1.25, 1.5, 1.75, 2.0, 3.0, 4.0, 5.0, 10.0,
```

Comment:
May be we dont need the last one. 10 sec per blk seems too long and we do have per-block gas limit change on the way.

```suggestion
    0.20, 0.30, 0.40, 0.50, 0.60, 0.70, 0.80, 0.90, 1.0, 1.25, 1.5, 1.75, 2.0, 3.0, 4.0, 5.0,
```